configs/nextflow.hopper.config

```diff
@@ -0,0 +1,170 @@
+// Hopper specific configuration for micro wgs pipeline //
+
+singularity{
+  enabled = true
+  runOptions = '--bind /fs1/ --bind /local/'
+}
+
+process {
+	executor='slurm'
+	queue='grace-high'
+	time=2.h
+
+	withLabel: 'process_high' {
+		cpus = 4
+		memory = '16 GB'
+		time = '4h'
+	}
+	withLabel: 'process_medium' {
+		cpus = 2
+		memory = '8 GB'
+		time = '1h'
+	}
+	withLabel: 'process_low' {
+		cpus = 1
+		memory = '1 GB'
+		time = '1h'
+	}
+
+	withName: ariba_run {
+		container = "/fs1/pipelines/JASEN/container/ariba.sif"
+		memory = '32 GB'
+		publishDir = "/fs1/results_dev/jasen/ariba"
+	}
+	withName: ariba_summary {
+		container = "/fs1/pipelines/JASEN/container/ariba.sif"
+		publishDir = "/fs1/results_dev/jasen/ariba"
+	}
+	withName: 'bwa_mem|bwa_index' {
+		container = "/fs1/pipelines/JASEN/container/bwa.sif"
+		cpus = 16
+		memory = '32 GB'
+		time = '1h'
+		publishDir = "/fs1/results_dev/jasen/bwa"
+	}
+	withName: 'samtools_sort|samtools_index' {
+		container = "/fs1/pipelines/JASEN/container/samtools.sif"
+		publishDir = "/fs1/results_dev/jasen/samtools"
+	}
+	withName: spades {
+		container = "/fs1/pipelines/JASEN/container/spades.sif"
+		cpus = 16
+		memory = '64 GB'
+		time = '2h'
+		publishDir = "/fs1/results_dev/jasen/spades"
+	}
+	withName: quast {
+		container = "/fs1/pipelines/JASEN/container/quast.sif"
+		publishDir = "/fs1/results_dev/jasen/quast"
+	}
+	withName: mlst {
+		container = "/fs1/pipelines/JASEN/container/mlst.sif"
+		publishDir = "/fs1/results_dev/jasen/mlst"
+	}
+	withName: kraken {
+		cpus = 16
+		memory = '48 GB'
+		time = '2h'
+		container = "/fs1/pipelines/JASEN/container/kraken2.sif"
+		publishDir = "/fs1/results_dev/jasen/kraken"
+	}
+	withName: bracken {
+		cpus = 16
+		memory = '48 GB'
+		time = '2h'
+		container = "/fs1/pipelines/JASEN/container/bracken.sif"
+		publishDir = "/fs1/results_dev/jasen/kraken"
+	}
```

Comment:
Quite over the top in resources. In general the parameters should be refined, but that can come in a later version, so this is mainly a n.b.

Reply:
This was Markus' recommendation. What would you suggest?

Reply:
I changed it to a quarter of the resources